### ISSUE TITLE
HDMI cutouts for mellow fly LCD would've been nice

### ISSUE THREAD

**Issue by yuriy-yarosh** [VzBoT3D/VzBoT-Vz330#64]
There's a HDMI + jack cutout missing in the mellow LCD [back  cover](https://github.com/VzBoT3D/VzBoT-Vz330/blob/master/Assemblies%20BOM%20and%20STL/electronics/Screen%20Mount/Mellow-Fly%20Version/Back%20cover.stl) mount STL.
Front panel needs a bit of a wider cutout as well.
Maybe worth making a separate HDMI version ?

![зображення](https://github.com/VzBoT3D/VzBoT-Vz330/assets/6811933/6f41c819-07ab-48cd-89c1-743840c2abd6)

**Comment by HumbleDeer** [comment]
Is this a different display board from the one the case was made for?

**Comment by yuriy-yarosh** [comment]
@HumbleDeer No, it's just the display board has multiple adapters for various interfaces.
Mellow vzbot kits are shipped with a DSI adapter for rPI, but DSI is somewhat rPI exclusive, so it make sense to replace it with a HDMI+USB to FRC adapter (flex ribbon) instead, e.g. for Mellow Fly PI.
